torch/_inductor/codegen/triton.py

```diff
@@ -1439,7 +1440,7 @@ def init_cooperative_reduction(self):
             if tree.grid_dim is not None:
                 tree.grid_dim += 1
 
-        sem_count, _ = self.numels
+        sem_count, _ = self.numels.values()
```

Comment:
This line should be updated to do a dict access to the `x` dim.

Reply:
Updated [d3146d1](https://github.com/pytorch/pytorch/pull/141751/commits/d3146d175c2a6ebbeff582d1561c2539ea1cf91e)